exercises/acronym/test/Tests.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import Acronym (abbreviate)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "acronym" $
+          describe "abbreviate" $ for_ cases test
+  where
+    test Case {..} = it description $ abbreviate input `shouldBe` expected
+
+-- Test cases adapted from
+-- `exercism/x-common/exercises/acronym/canonical-data.json`
+-- on 2017-03-31
```

Comment:
same q as https://github.com/exercism/xhaskell/pull/511#discussion_r109295131 - how should we put the version in here?